tests/suites/test_suite_ccm.data

```diff
@@ -14,8 +14,8 @@ depends_on:MBEDTLS_AES_C
 mbedtls_ccm_setkey:MBEDTLS_CIPHER_ID_AES:224:MBEDTLS_ERR_CCM_BAD_INPUT
 
 CCM init #4 BLOWFISH-128: bad block size
-depends_on:MBEDTLS_BLOWFISH_C
-mbedtls_ccm_setkey:MBEDTLS_CIPHER_ID_BLOWFISH:128:MBEDTLS_ERR_CCM_BAD_INPUT
+depends_on:MBEDTLS_DES_C
```

Comment:
Please fix the description on the previous line too, as it's still mentioning blowfish.